src/buffer/out/textBuffer.cpp

```diff
@@ -1375,25 +1377,21 @@ til::point TextBuffer::_GetWordEndForAccessibility(const til::point target, cons
     }
     else
     {
-        auto iter{ GetCellDataAt(result, bufferSize) };
-        while (iter && iter.Pos() != limit && _GetDelimiterClassAt(iter.Pos(), wordDelimiters) == DelimiterClass::RegularChar)
+        while (result != limit && result != bufferSize.BottomRightInclusive() && _GetDelimiterClassAt(result, wordDelimiters) == DelimiterClass::RegularChar)
         {
             // Iterate through readable text
-            ++iter;
+            bufferSize.IncrementInBounds(result);
         }
 
-        while (iter && iter.Pos() != limit && _GetDelimiterClassAt(iter.Pos(), wordDelimiters) != DelimiterClass::RegularChar)
+        while (result != limit && result != bufferSize.BottomRightInclusive() && _GetDelimiterClassAt(result, wordDelimiters) != DelimiterClass::RegularChar)
         {
             // expand to the beginning of the NEXT word
-            ++iter;
+            bufferSize.IncrementInBounds(result);

```

Comment:
I've been meaning to review this, but I still can't get to this. I'm still hung-up on fixing all remaining bugs that I can find before we ship the next big version of this project with Windows.

From skimming it, I believe that this PR is mostly correct, but I'm somewhat worried that it goes into the wrong direction. In particular, there's this line. This one doesn't properly work with wide glyphs (= 2 column wide glyphs like Chinese/Japanese), whereas the previous code probably did it correctly from what I can tell. If you get a chance, I'd suggest testing this PR with such characters.

To be clear, we'll definitely review this PR in time for the next version of Windows Terminal, that is planned to first ship in Preview sometime early next year, but it's possible that a proper review may be delayed until then.